wemake_python_styleguide/visitors/ast/naming.py

```diff
@@ -75,7 +79,10 @@ def visit_Attribute(self, node: ast.Attribute) -> None:
 
         self.generic_visit(node)
 
-    def visit_FunctionDef(self, node: ast.FunctionDef) -> None:
+    def visit_FunctionDef(
+        self,
+        node: Union[ast.FunctionDef, ast.AsyncFunctionDef],
```

Comment:
We already have a type for it: https://github.com/wemake-services/wemake-python-styleguide/blob/3cedeb3c13ab6b16980a39edf657ab93d4c1f19e/wemake_python_styleguide/visitors/ast/complexity/counts.py#L20

Can you please move it to `types.py` and reuse it?
A comment docstring will also be required.